config/webpack.dev.config.js

```diff
@@ -69,5 +70,13 @@ module.exports = Merge.smart(commonConfig, {
         pathRewrite: { '^/api': '' },
       },
     },
+    setup(app) {
+      app.use(errorOverlayMiddleware());
+    },
   },
+  entry: [
+    require.resolve('react-dev-utils/webpackHotDevClient'),
+    require.resolve('react-error-overlay'),
+    path.resolve(__dirname, '../src/index.jsx'),
```

Comment:
I think this line is unnecessary because this will be merged with the entry array in [`webpack.common.config.js`](https://github.com/edx/studio-frontend/blob/69c5a19737a9b65e90837ebadb80e42291a937c7/config/webpack.common.config.js#L7).

Reply:
Oh wait, that's not an array. Maybe it won't merge, not sure actually...

Reply:
Nope, Merge can't handle an object + an array. It actually sort of works merging with the following in webpack.dev.config:
```
  entry: {
    hotDev: require.resolve('react-dev-utils/webpackHotDevClient'),
  },
```

But that makes the hot reloader occasionally fill up the call stack for some reason :/